samples/sampler/package.json

```diff
@@ -31,11 +31,11 @@
     "react": "^17.0.1"
   },
   "devDependencies": {
-    "@enact/storybook-utils": "^2.0.0",
-    "@storybook/addons": "5.3.5",
-    "@storybook/react": "5.3.5",
-    "@storybook/theming": "5.3.5",
-    "babel-loader": "^8.0.6",
-    "core-js": "^3.1.4"
+    "@enact/storybook-utils": "enactjs/storybook-utils#feature/react17",
```

Comment:
He we'll need mikyung's opinion regarding the version

Reply:
Since the 4.0.0 version of storybook-utils was released, we can use it